doc/release/upcoming_changes/15870.new_feature.rst

```diff
@@ -0,0 +1,11 @@
+``keepdims`` parameter for `numpy.count_nonzero`
+------------------------------------------------
+The parameter ``keepdims`` was added to `numpy.count_nonzero`. The
+parameter has the same meaning as it does in reduction functions such
+as `numpy.sum` or `numpy.mean`.  When an ``axis`` argument is given,
+the slice along that dimension (or dimensions, if ``axis`` is a tuple
+with length greater than one) is reduced to a single number in the
+output.  Normally this means the shape of the output array has fewer
+dimensions than the input.  When ``keepdims=True`` is used, the output
+array has the same number of dimensions as the input, and the lengths
+of the reduced dimensions are one.
```

Comment:
Not sure if this section is needed, the user can look at the docs if they want the meaning.

Will let the person merging decide

Reply:
Thanks @eric-wieser. I don't mind paring down the release note, but I'll wait for more feedback before copy-editing.

Reply:
On second thought, I agree with the suggestion, so I just cut the release note down to the first two sentences.